### DISCUSSION TITLE
Does FeatherPad change line endings to Unix on save?

### DISCUSSION THREAD

**Discussion by ghost** [tsujan/FeatherPad#820] [Q&A] [answered]
I'm using MX Linux. FeatherPad is the default text viewer. My notes are created on Windows with CRLF line endings. I did some editing in MX Linux and, when I'm back to Windows, I realized all line endings are changed to Unix.

**Comment by tsujan** [comment, marked as answer]
> Does FeatherPad change line endings to Unix on save?

Yes, as Qt does.

If you want MS Windows end-of-lines, use `File → Save with Encoding`; it'll show a prompt dialog about that.

**Reply by ghost** [reply]
There is no global config in `Preferences`. I don't want to do this with each saves!

**Reply by tsujan** [reply]
Sorry, I have no plan to add an option for it.

Recent versions of Windows should have no problem with it, like other OS's don't. As README says, FeatherPad is an editor for Linux.
